pipelines/ci-build.yml

```diff
@@ -298,17 +298,3 @@ extends:
                     packagesToPush: '$(Pipeline.Workspace)\Microsoft.Graph.Beta.*.nupkg'
                     nuGetFeedType: external
                     publishFeedCredentials: 'microsoftgraph NuGet connection'
-                - task: GitHubRelease@1
```

Comment:
Before this step stopped working it added the nuget packages to the release in GitHub, e.g. https://github.com/microsoftgraph/msgraph-beta-sdk-dotnet/releases/tag/5.117.0-preview

Do we know why this stopped working? Can we resolve the underlying issue?

If we can't then removing this task is fine by me, but I'd like to make sure we understand what the problem is first.